docs/application/dotnet/tutorials/create-dotnet-hybrid-package.md

```diff
@@ -39,37 +40,37 @@ $[TIZEN_STUDIO_DIR]/tools/ide/bin$ ./tizen package -t wgt -s [CERTIFICATE_PROFIL
 
     ![Create Web Widget 4](./media/create_web_sample_cs_4.png)
 
-2. Check manifest file of web application
+2. Check the **config.xml** file of the Web application.
```

Comment:
Will be changing it to colon instead of a period in the next commit.

```suggestion
2. Check the **config.xml** file of the Web application:
```

Reply:
Noted!

Reply:
Thank you.